apps/01PigLatin.js

```diff
@@ -9,8 +9,42 @@ var rl = readline.createInterface({
 
 
 function pigLatin(word) {
-
+word=word.toLowerCase();
   // Your code here
+  console.log(word[0]);
+  var firstletter=word[0];
+
+  var vowelIndex = -1; // Set it to assume there are no vowels
+
+if ( ( word.indexOf('a') > -1 && word.indexOf('a') < vowelIndex ) || vowelIndex === -1 ) {
+    vowelIndex = word.indexOf('a');
+}
+
+if ( ( word.indexOf('e') > -1 && word.indexOf('e') < vowelIndex ) || vowelIndex === -1 ) {
+    vowelIndex = word.indexOf('e');
+}
+
+if ( ( word.indexOf('i') > -1 && word.indexOf('i') < vowelIndex ) || vowelIndex === -1 ) {
+    vowelIndex = word.indexOf('i');
+}
+
+if ( ( word.indexOf('o') > -1 && word.indexOf('o') < vowelIndex ) || vowelIndex === -1 ) {
+    vowelIndex = word.indexOf('o');
+}
+
+if ( ( word.indexOf('u') > -1 && word.indexOf('u') < vowelIndex ) || vowelIndex === -1 ) {
+    vowelIndex = word.indexOf('u');
+}
+
+var firstPart = word.slice(0, vowelIndex);
+var restWord = word.slice(vowelIndex, word.length);
+
+if(vowelIndex ===0){
+  return word + 'yay'
+}
+
+return restWord + firstPart + 'ay';
+
 
```

Comment:
Your indentation seems a little wonky - there's an easy fix I posted on discourse should help you pass the CircleCi tests.